bin/prover-client/src/operators/constants.rs

```diff
@@ -0,0 +1,6 @@
+/// Maximum number of blocks allowed in a proving range.
+///
+/// This constant serves as a safety limit to prevent proving tasks from processing
+/// an excessively large number of blocks. If the number of blocks to prove exceeds
+/// this limit, the task will be aborted early.
+pub const MAX_PROVING_BLOCK_RANGE: u64 = 1024;
```

Comment:
Does it make sense it to be passed as an argument? I don't have a strong opinion on this. Also let's add a rationale for using `1024` instead of other number.